### DISCUSSION TITLE
Hardlinked files show different inodes ?

### DISCUSSION THREAD

**Discussion by LelouBil** [trapexit/mergerfs#1339] [Q&A] [answered]
Hello, 
I'm trying out mergerfs as a replacement for LVM for my media server, and I am trying hardlinks.
It works, hardlinks on the mergerfs filesystem do actual hardlinks on the underlying disk, however the inodes number presented by mergerfs are not the same for hardlinked files.

This seems to contradict the README here https://github.com/trapexit/mergerfs?tab=readme-ov-file#inodecalc

Even if I use devino-hash, different inodes are presented, I hesitate to file a bug since I saw this issue https://github.com/trapexit/mergerfs/issues/277, however the README seems to say the contrary.

**Comment by trapexit** [comment, marked as answer]
Please provide the information as requested in the ticket and the support section of the docs.

![image](https://github.com/trapexit/mergerfs/assets/197676/853bf7b9-4dde-4915-b85a-36b1e58f627d)

**Reply by LelouBil** [reply]
Thanks for pointing me to this, I was using an old version from the debian repos, and it's fixed in latest
